vcluster/learn-how-to/control-plane/container/safely-upgrade-etcd.mdx

```diff
@@ -0,0 +1,81 @@
+
+etcd has identified and fixed an additional scenario that may cause upgrade failures when moving from etcd v3.5 to v3.6, specifically during rolling replacement upgrades where a new learner may receive an incorrect snapshot from an older member (≤ v3.5.19). This issue, detailed in [issue #20793](https://github.com/etcd-io/etcd/issues/20793), was addressed in etcd v3.5.24 via [PR #20797](https://github.com/etcd-io/etcd/pull/20797). <br/>
+The recommended solution is to first upgrade to etcd v3.5.24 (or higher) before proceeding to v3.6. For users on v3.5.20-v3.5.23 from a fresh install, a direct upgrade to v3.6 is safe. <br/>
+To safely upgrade etcd in vCluster, continue reading this page for detailed steps.<br/><br/>
+For more details, refer to the [etcd upgrade guide](https://etcd.io/docs/latest/upgrades/upgrade_3_6/).
+
+<BackingStore />
+
+## Version matrix for embedded ETCD and vCluster
+
+| vCluster Version | ETCD Version |
+|------------------|--------------|
+| v0.26.4          | v3.5.21      |
+| v0.26.5          | v3.5.24      |
+| v0.27.2          | v3.5.21      |
+| v0.27.3          | v3.5.24      |
+| v0.28.1          | v3.5.21      |
+| v0.28.2          | v3.5.24      |
+| v0.29.*+         | v3.6.*       |
+
+**Important**: Before upgrading, read the [ETCD blog post about upgrading from 3.5 to 3.6](https://etcd.io/blog/2025/upgrade_from_3.5_to_3.6_issue_followup/).
+
+<Tabs>
+  <TabItem value="embedded" label="Embedded ETCD">
+
+    <Tabs>
+      <TabItem value="v0.26" label="vCluster v0.26.[0-4]">
+        First upgrade to v0.26.5, it uses ETCD v3.5.24. Then, your ETCD data will be saved in a format compatible with upgrade to v3.6
+      </TabItem>
+      <TabItem value="v0.27" label="vCluster v0.27.[0-2]">
+        First upgrade to v0.27.3, it uses ETCD v3.5.24. Then, your ETCD data will be saved in a format compatible with upgrade to v3.6
+      </TabItem>
+      <TabItem value="v0.28" label="vCluster v0.28.[0-1]">
+        First upgrade to v0.28.2, it uses ETCD v3.5.24. Then, your ETCD data will be saved in a format compatible with upgrade to v3.6
+      </TabItem>
+    </Tabs>
+
+  </TabItem>
+
+  <TabItem value="deploy" label="Deployed ETCD">
+
+    Follow the ETCD-recommended upgrade path:
+    1. **v3.5.21 → v3.5.24**
+    2. **v3.5.24 → v3.6.x**
+
+    You may set ETCD image tag via:
+    ```yaml title="vcluster.yaml"
+    controlPlane:
+      backingStore:
+        etcd:
+          deploy:
+            statefulSet:
+              image:
+                registry: quay.io
+                repository: coreos/etcd
+                tag: v3.5.24
+    ```
+
+  </TabItem>
+
+  <TabItem value="external" label="External ETCD">
+
+    For external ETCD configurations, vCluster does not manage the ETCD deployment. You are responsible for following the [ETCD upgrade procedures](https://etcd.io/docs/v3.6/upgrades/upgrade_3_6/) according to your infrastructure setup.
+
+  </TabItem>
+</Tabs>
```

Comment:
We have a big "Upgrade Notice" warning here: https://www.vcluster.com/docs/vcluster/next/configure/vcluster-yaml/control-plane/components/backing-store/etcd/embedded
Can you make that one smaller and link out to this new page.